test/functional/wallet_resendwallettransactions.py

```diff
@@ -64,6 +64,9 @@ def run_test(self):
         # Transaction should be rebroadcast approximately 24 hours in the future,
         # but can range from 12-36. So bump 36 hours to be sure.
         node.setmocktime(now + 36 * 60 * 60)
+        # Give some time for trickle to occur
+        time.sleep(1)
```

Comment:
It seems a bit confusing to "use mockable time everywhere", but then start adding wallclock sleeps in tests. Also, the comment refers to "trickle", but should probably say "wallet resend". (The trickle happens when you call `setmocktime + 600` in the next line). I suggest the following diff:

```diff
diff --git a/test/functional/wallet_resendwallettransactions.py b/test/functional/wallet_resendwallettransactions.py
index 9336f94ee3..f2f48b8bcd 100755
--- a/test/functional/wallet_resendwallettransactions.py
+++ b/test/functional/wallet_resendwallettransactions.py
@@ -64,8 +64,8 @@ class ResendWalletTransactionsTest(BitcoinTestFramework):
         # Transaction should be rebroadcast approximately 24 hours in the future,
         # but can range from 12-36. So bump 36 hours to be sure.
         node.setmocktime(now + 36 * 60 * 60)
-        # Give some time for trickle to occur
-        time.sleep(1)
+        # Tell scheduler to call MaybeResendWalletTxs now
+        node.mockscheduler(1)
         node.setmocktime(now + 36 * 60 * 60 + 600)
         peer_second.wait_for_broadcast([txid])

Reply:
Done.